package/doc/sphinx/source/documentation_pages/analysis/hole2.rst

```diff
@@ -1,35 +1,282 @@
-===================================================
- HOLE analysis --- :mod:`MDAnalysis.analysis.hole2`
-===================================================
+"""HOLE Analysis --- :mod:`MDAnalysis.analysis.hole2`
+=====================================================================================
```

Comment:
This shows up as extra quote marks in the built docs. Please have a look at the rendered documentation here: https://mdanalysis--3579.org.readthedocs.build/en/3579/documentation_pages/analysis/hole2.html

Also, the `====` line is too long, although sphinx seems to have ignored it.

This is part of the "ReadTheDocs" check that is in the bottom box in the pull request. Whenever you push to this branch, that check will run and update the link I pasted in the comment. It might take some time if other people are also working on MDAnalysis, but if that is green, the documentation will have updated.

Please check this link every time you update the documentation.

<img width="916" alt="Screen Shot 2022-04-06 at 9 06 56 am" src="https://user-images.githubusercontent.com/31115101/161865314-1b97d58c-3845-4c6f-b5fa-6e8427b3aeb6.png">